### PR TITLE
Begin renaming project from iModel.js to iTwin.js

### PR DESCRIPTION
**Pull request by kabentley**
The begins the process of renaming our project from `iModel.js` to `iTwin.js` to better reflect its purpose as the platform for **infrastructure digital twins**. 

iModels are of course a big part of iTwins, so much of the api remains iModel-centric. But, the many parts that don't have a direct relationship to iModels will use the term *iTwin* going forward to avoid confusion.

The full conversion will be made gradually and incrementally, and will likely take several major release cycles to complete. This PR begins with redirecting `www.imodeljs.org` to `www.itwinjs.org`, and updating references to the project name in markdown files.

### CODE REVIEW

**Review by bbastings — APPROVED** [review]

**Review by bsteinbk — APPROVED** [review]

File: docs/getting-started/index.md [hunk 1]
```diff
@@ -38,9 +38,9 @@ The starter application is located in the [desktop-starter](https://github.com/i
 
 > Open your snapshot iModel in the desktop-starter app by clicking the home button in the upper left and clicking "Select snapshot".
 
-### [Follow tutorials to dive deeper into iModel.js]($docs/learning/tutorials/index.md)
+### [Follow tutorials to dive deeper into iTwin.js]($docs/learning/tutorials/index.md)
 
-### [iModel.js Blog](https://medium.com/imodeljs)
+### [iTwin.js Blog](https://medium.com/imodeljs)
```
Comment (calebmshafer) [inline docs/getting-started/index.md:43]: Can you update the blob link?  It's now at `https://medium.com/itwinjs`

File: docs/getting-started/online-quickstart.md [hunk 2]
```diff
@@ -51,9 +51,9 @@ Edit imodeljs-samples/interactive-app/basic-viewport-app/.env.local
 
 ## Next Steps
 
-### [Follow tutorials to dive deeper into iModel.js]($docs/learning/tutorials/index.md)
+### [Follow tutorials to dive deeper into iTwin.js]($docs/learning/tutorials/index.md)
 
-### [iModel.js Blog](https://medium.com/imodeljs)
+### [iTwin.js Blog](https://medium.com/imodeljs)
```
Comment (calebmshafer) [inline docs/getting-started/online-quickstart.md:56]: Here as well.

**Review by calebmshafer — CHANGES_REQUESTED** [review]
Do we want to add the explanation you gave in the PR description somewhere in the docs themselves?  That way if people see a mix of iTwin and iModel, it'll hopefully reduce confusion.

File: docs/getting-started/online-quickstart.md (hunk 2 above)
Comment (kabentley) [reply docs/getting-started/online-quickstart.md:56]: @calebmshafer how about the Twitter feed in `CommunityResources.md`? Has that been changed?

File: docs/getting-started/index.md (hunk 1 above)
Comment (kabentley) [reply docs/getting-started/index.md]: done

File: docs/getting-started/online-quickstart.md (hunk 2 above)
Comment (calebmshafer) [reply docs/getting-started/online-quickstart.md]: We don't currently have the itwinjs or itwin Twitter feeds, just imodeljs so we probably shouldn't switch that one yet.

Comment (calebmshafer) [reply docs/getting-started/online-quickstart.md]: Just sent a message to a few people about locking down one, or both, of those handles.

**Review by calebmshafer — APPROVED** [review]

**Review by pmconne — APPROVED** [review]

**Review by saskliutas — APPROVED** [review]

**Review by diegoalexdiaz — APPROVED** [review]

**Review by mdastous-bentley — APPROVED** [review]

**Review by williamkbentley — APPROVED** [review]

**Review by wgoehrig — APPROVED** [review]

**Pull request merged**